src/lib/storage-utils.ts

```diff
@@ -236,3 +236,44 @@ export function isPublishablePackage(pkg: Package): boolean {
 
   return _.includes(keys, 'versions');
 }
+
+export function convertAbbreviatedManifest(manifest: Manifest): AbbreviatedManifest {
+  const abbreviatedVersions = Object.keys(manifest.versions).reduce((acc: AbbreviatedVersions, version: string) => {
+    const _version = manifest.versions[version];
+    // This should be align with this document
+    // https://github.com/npm/registry/blob/master/docs/responses/package-metadata.md#abbreviated-version-object
```

Comment:
looks like that document is incorrect. os and cpu are also included in the abbreviated version